src/applications/financial-status-report/components/Typeahead.jsx

```diff
@@ -89,7 +92,7 @@ const Typeahead = ({ uiSchema, idSchema, formData, onChange, onBlur }) => {
       const fetchedsuggestions = getSuggestions(input);
       setSuggestions(fetchedsuggestions);
     },
-    [input, getInput, getSuggestions],
+    [], // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
this pr [Requested upgrade: React Redux #16419](16419) is causing an extra render for the FSR Typeahead component and is a breaking change so removing dependencies to avoid future issues